### ISSUE TITLE
Summary of all trained models?

### ISSUE THREAD

**Issue by holyseven** [zhengyang-wang/Deeplab-v2--ResNet-101--Tensorflow#23]
Could you summarize a little on the trained models with different configurations and different databases? For example, for pascal voc 2012 validation set, pre-trained models (resnet50/resnet101/deeplab), training with/without msc, evaluate with/without msc.

(by the way, the difference between two pre-trained models, resnet101 and deeplab, is just the resnet101 is pre-trained on ImageNet, and deeplab is pre-trained on ImageNet and COCO?)

**Comment by zhengyang-wang** [comment]
ResNet50 and ResNet101 are simply pre-trained on ImageNet.
For deeplab pre-trained models, deeplab_resnet_init.ckpt is pre-trained on ImageNet and MSCOCO while deeplab_resnet.ckpt is pre-trained on ImageNet, MSCOCO and PASCAL VOC (both train and val). And they used msc when performing pre-training on MSCOCO and PASCAL VOC.

(Yes)

**Comment by John1231983** [comment]
One more thing, the resnet pretrained achieved about 70.5% mIoU, while deeplab pretrained (imagenet+mscoco) obtained 75.4%. That is performance I achived

**Comment by zhengyang-wang** [comment]
71.7 (change learning rate to 0.001 as suggested in the paper) & 75.1 here.
